### PR TITLE
Objects Sorting

### PR DESCRIPTION
**Pull request by mohitpubnub**
Objects sorting code

### CODE REVIEW

**Comment by crimsonred** [comment]
@mohitpubnub please don't close the reviewed PRs (as you did https://github.com/pubnub/unity/pull/50). I will have to review each file again, that rework that can be avoided.

**Comment by mohitpubnub** [comment]
> @mohitpubnub please don't close the reviewed PRs (as you did #50). I will have to review each file again, that rework that can be avoided.

Noted!
Just to summarize- The changes requested are:
1. Sort fields should be List<string> instead array of string
2. Sort query param value should be encoded which I haven't
3. Enhance Integration Tests- It should be verified that it's actually sorting -> For that ascending and descending sort tests are added for that same data to verify it

File: .pubnub.yml [hunk 1]
```diff
@@ -1,5 +1,12 @@
 ---
```
Comment (crimsonred) [inline .pubnub.yml:1]: Add OBJECTS-SORTING in feature matrix

**Comment by crimsonred** [comment]
Integration test for SORT failing: https://travis-ci.org/github/pubnub/unity/builds/670184081#L1894-L1918 @mohitpubnub

**Comment by mohitpubnub** [comment]
> Integration test for SORT failing: https://travis-ci.org/github/pubnub/unity/builds/670184081#L1894-L1918 @mohitpubnub

I'm on it!!

File: .pubnub.yml (hunk 1 above)
Comment (mohitpubnub) [reply .pubnub.yml:1]: Done!

**Comment by mohitpubnub** [comment]
> > Integration test for SORT failing: https://travis-ci.org/github/pubnub/unity/builds/670184081#L1894-L1918 @mohitpubnub
> 
> I'm on it!!

Should(hopefully🤞) Pass now! `MangeMembers` & `ManageMemberships` had missing params

**Comment by mohitpubnub** [comment]
> Did you test with all the platforms that we support: iOS, Android, Windows, Windows Store Universal 10, Mac, Linux and WebGL?
> 
> @mohitpubnub

Not yet! Wasn't aware of that fact!
How Can I do that? I mean, Where can I find ideal environments for Unity SDK where we are supposed to test.

**Comment by mohitpubnub** [comment]
@crimsonred 
[Build #616](https://travis-ci.org/github/pubnub/unity/builds/670450585?utm_medium=notification&utm_source=github_status)

SortTests Passing but
Failing at `TestMembersAndMemberships()` -> Create User

Not finding restart build option in my travis view!!

**Comment by crimsonred** [comment]
> > Did you test with all the platforms that we support: iOS, Android, Windows, Windows Store Universal 10, Mac, Linux and WebGL?
> > @mohitpubnub
> 
> Not yet! Wasn't aware of that fact!
> How Can I do that? I mean, Where can I find ideal environments for Unity SDK where we are supposed to test.

iOS = iOS Device, 
Android = Android Device, 
Windows / Windows Store Universal 10 = Windows machine/AppVeyor, 
Mac = Mac/Travis, 
Linux = Linux Machine 
WebGL = WebBrowser

**Comment by mohitpubnub** [comment]
> > > Did you test with all the platforms that we support: iOS, Android, Windows, Windows Store Universal 10, Mac, Linux and WebGL?
> > > @mohitpubnub
> > 
> > 
> > Not yet! Wasn't aware of that fact!
> > How Can I do that? I mean, Where can I find ideal environments for Unity SDK where we are supposed to test.
> 
> iOS = iOS Device,
> Android = Android Device,
> Windows / Windows Store Universal 10 = Windows machine/AppVeyor,
> Mac = Mac/Travis,
> Linux = Linux Machine
> WebGL = WebBrowser
@crimsonred 
Ran tests on build support platforms (downloaded through UnityHub 2018.4.16f1). Passed!
AppVeyor is remaining thing here! Which hasn't triggered yet.
Not sure, Am I supposed to add this barnch name to appveyor.yml (branches section)??

**Comment by crimsonred** [comment]
> AppVeyor is remaining thing here! Which hasn't triggered yet.

It's configured to run on develop and master. You don't need to do anything.

**Pull request merged**